client/src/reducers/index.ts

```diff
@@ -0,0 +1,4 @@
+import { createStore } from "redux";
+import { reducer } from "./Reducer";
+
+export const store = createStore(reducer);
```

Comment:
CreateStore takes in reducer - the current reducer is only Page1Reducer.. But what happens when we have many reducers? We should use **combineReducers** as shown above.

Reply:
@johnagl oh I missed changing this, my bad. It took me a while to implement combineReducers.